### PR TITLE
Pre release

### PR DESCRIPTION
**Pull request by Lagovas**
Updated changelog and versions for new 0.94.0 release.

## Checklist

- [ ] Change is covered by automated tests
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation in the [Acra documentation] site or has PR on [documentation repository] 
  with new changes
- [ ] CHANGELOG.md is updated (in case of notable or breaking changes)
- [ ] CHANGELOG_DEV.md is updated
- [ ] Benchmark results are attached (if applicable)
- [ ] [Example projects and code samples] are up-to-date (in case of API changes) 

[coding guidelines]: https://golang.org/doc/effective_go
[Example projects and code samples]: https://github.com/cossacklabs/acra-engineering-demo
[Acra documentation]: https://docs.cossacklabs.com/
[documentation repository]: https://github.com/cossacklabs/product-docs

### CODE REVIEW

**Review by Zhaars — APPROVED** [review]
Seems like everything is here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+  - In-memory cache for keystore now uses randomly generated symmetric key for key encryption instead of ACRA_MASTER_KEY. [#555](https://github.com/cossacklabs/acra/pull/555)
+  - Improved reloading on SIGHUP signals. [#557](https://github.com/cossacklabs/acra/pull/557)
+- **AcraServer**: 
+  - Supports [HashiCorp Consul](https://www.consul.io/) as configuration source for encryptor config. Acra can load configuration from the Consul instead of file. Were added new CLI flag (`--encryptor_config_storage_type=[filesystem|consul]`) to switch source and Consul specific flags. Read more on [documentation page](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config/) about encryptor config and acra-server's [configuration description](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-server/#hashicorp-consul). [#568](https://github.com/cossacklabs/acra/pull/568)
+  - Supports searchable tokenization. AcraServer captures `SELECT` queries and update `WHERE` clauses to add support of filtering with consistent tokenization. [#581](https://github.com/cossacklabs/acra/pull/581)
```
Comment (vixentael) [inline CHANGELOG.md:26]:
```suggestion
  - Improved support of searchable tokenization. AcraServer captures `SELECT` queries and update `WHERE` clauses to add support of filtering with consistent tokenization. [#581](https://github.com/cossacklabs/acra/pull/581)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+  - Improved searchable encryption with more complex queries. [#586](https://github.com/cossacklabs/acra/pull/586), [#592](https://github.com/cossacklabs/acra/pull/592), [#598](https://github.com/cossacklabs/acra/pull/598), [#599](https://github.com/cossacklabs/acra/pull/599), [#594](https://github.com/cossacklabs/acra/pull/594).
+  - Improved sqlparser:
```
Comment (vixentael) [inline CHANGELOG.md:28]:
```suggestion
  - Improved SQL parser (better compatibility across different SQL databases):
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +41,17 @@ _Core_:
       - Table names are case-insensitive by default, could be changed with `case_sensitive_table_identifiers` option mentioned above
     Case-insensitive means the identifier is converted to lowercase before comparing with values from encryptor config, encryptor config should contain lowercase version of column/table name.
     Case-sensitive means identifiers are compared with values from encryptor config "as is", encryptor config should contain exactly the same identifier as in database schema.
+  - Removed deprecated `--tls_db_sni` flag. Now only `--tls_database_sni` is available. [#564](https://github.com/cossacklabs/acra/pull/564)
+  - Supports separate configuration and specifying of CRL/OCSP settings for connections from database and applications. Were added flags: `--tls_ocsp_[database|client]_required`, `--tls_[ocsp|crl]_[database|client]_check_only_leaf_certificate`, `--tls_[ocsp|crl]_[database|client]_from_cert`, `--tls_[ocsp|crl]_[database|client}_cache_size`, `--tls_[ocsp|crl]_[database|client}_cache_time`, `--tls_[ocsp|crl]_[database|client}_cache_size` [#564](https://github.com/cossacklabs/acra/pull/564).
```
Comment (vixentael) [inline CHANGELOG.md:45]:
```suggestion
  - Added support of separate configuration and specifying of CRL/OCSP settings for connections from database and applications. Added flags: `--tls_ocsp_[database|client]_required`, `--tls_[ocsp|crl]_[database|client]_check_only_leaf_certificate`, `--tls_[ocsp|crl]_[database|client]_from_cert`, `--tls_[ocsp|crl]_[database|client}_cache_size`, `--tls_[ocsp|crl]_[database|client}_cache_time`, `--tls_[ocsp|crl]_[database|client}_cache_size` [#564](https://github.com/cossacklabs/acra/pull/564). 
```

Comment (vixentael) [reply CHANGELOG.md:45]: maybe link with docs here as well?

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,6 +41,17 @@ _Core_:
...
     Case-sensitive means identifiers are compared with values from encryptor config "as is", encryptor config should contain exactly the same identifier as in database schema.
+  - Removed deprecated `--tls_db_sni` flag. Now only `--tls_database_sni` is available. [#564](https://github.com/cossacklabs/acra/pull/564)
+  - Supports separate configuration and specifying of CRL/OCSP settings for connections from database and applications. Were added flags: `--tls_ocsp_[database|client]_required`, `--tls_[ocsp|crl]_[database|client]_check_only_leaf_certificate`, `--tls_[ocsp|crl]_[database|client]_from_cert`, `--tls_[ocsp|crl]_[database|client}_cache_size`, `--tls_[ocsp|crl]_[database|client}_cache_time`, `--tls_[ocsp|crl]_[database|client}_cache_size` [#564](https://github.com/cossacklabs/acra/pull/564). 
+- **AcraTranslator**:
+  - Refactored HTTP processing core. Now AcraTranslator uses golang's [HTTP server](https://pkg.go.dev/net/http) with [gin](https://github.com/gin-gonic/gin) router [#550](https://github.com/cossacklabs/acra/pull/550). Was added support of:
```
Comment (vixentael) [inline CHANGELOG.md:47]:
```suggestion
  - Improved HTTP API performance. Refactored HTTP processing core. Now AcraTranslator uses golang's [HTTP server](https://pkg.go.dev/net/http) with [gin](https://github.com/gin-gonic/gin) router [#550](https://github.com/cossacklabs/acra/pull/550). Added support of:
```

**Review by vixentael — CHANGES_REQUESTED** [review]
fantastic!

please see my comments

**Review by vixentael — APPROVED** [review]

**Pull request merged**